src/main/java/com/team1678/frc2024/Constants.java

```diff
@@ -459,7 +443,7 @@ public static final class HoodConstants {
 
 			kHoodServoConstants.kHomePosition = 15.0; // Degrees
 			kHoodServoConstants.kRotationsPerUnitDistance = (1.0 / 360.0) * (7.16 / 1.0); // Cancoder to unit distance
-			kHoodServoConstants.kKp = isEpsilon ? 100 : 200;
+			kHoodServoConstants.kKp = 100; // 200
```

Comment:
We need to test 200, since that's what their final number was for their competition bot.